File: swagger/models/get_classes_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .class import Class

class GetClassesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		classes
		A list of the requested classes.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Classes': ['classes', Class],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'classes': 'Classes',
		}

	pagination_response = None
	classes = None

File: swagger/Models/GetClassesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClassesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse
	 */
	public $paginationResponse;
	/**
	 * A list of the requested classes.
	 * 
	 * @var Class[]
	 */
	public $classes;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class],
			'Classes' => ['classes', Class::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => 'PaginationResponse',
			'classes' => 'Classes',
			];
	}
}
